code/hispania/game/objects/items/weapons/kitchen.dm

```diff
@@ -0,0 +1,34 @@
+/obj/item/weapon/cheese_grater
```

Comment:
i love this recipe.. but its also not in the pr document

Reply:
want me to do it but in eris folder?